comment.php

```diff
@@ -0,0 +1,62 @@
+<?php include('config/init.php');
+$post = getPostInformation($_REQUEST['ID']);
+siteHeader('comments','blog + inspiration');
+
+$body=$_REQUEST['body'];
+$name=$_REQUEST['name'];
+$email=$_REQUEST['email'];
+$blogPostID=$_REQUEST['ID'];
+
+$result = createComment($body,$name,$email,$blogPostID);
```

Comment:
Whenever you're saving something from a form, it's best to accept the request and create the thing (like you're doing here) and the redirect to a new page that shows the results. What you're doing here works, but if the user reloads the page, it'll post the same comment a second time. The code should generally look something like this:

```
if(isset($_REQUEST['body'])){
   $commentId = createComment(...);
   header('location:whatever_file_you_want_to_redirect_to.php?newCommentId='.$commentId);
   exit;
}
```

So you receive the request, save the comment, and then redirect to a different page to show some confirmation. Note that I have `createComment` returning the id of the new comment we just created so that it can be included in the url of the page we're redirecting to.

Let me know if you want to discuss this more.